edx_event_bus_kafka/__init__.py

```diff
@@ -6,6 +6,7 @@
 See ADR ``docs/decisions/0006-public-api-and-app-organization.rst`` for the reasoning.
 """
 
+from edx_event_bus_kafka.internal.consumer import KafkaEventConsumer
```

Comment:
I know this is a draft PR, but I'm just now noticing that we have `EventProducerKafka` vs. `KafkaEventConsumer` -- probably want to get those names aligned.

Reply:
I like KafkaEventConsumer better. I'll change EventProducerKafka and up the version. Now that edx-event-bus-kafka is in normal dependencies it shouldn't try to bring in the upgrade before we're ready.